cmd/fs-v1.go

```diff
@@ -579,10 +579,29 @@ func (fs *FSObjects) DeleteBucket(ctx context.Context, bucket string, opts Delet
 	}
 
 	if !opts.Force {
+		configName := fmt.Sprintf(".sds-config.%s.json", bucket)
+		confPath := path.Join(bucketDir, configName)
+		tmpPath := fmt.Sprintf("/tmp/%s", configName)
```

Comment:
I opened an issue to test this on windows. Works on my mac.
Low priority for now but good to keep in mind through issues.